app/scripts/services/api.js

```diff
@@ -321,6 +321,8 @@ angular.module('bsis')
 
       MobileClinicLookUp: $resource(url + '/mobileclinic/lookup'),
 
-      DonationsReport: $resource(url + '/reports/collecteddonations/generate')
+      DonationsReport: $resource(url + '/reports/collecteddonations/generate'),
+
+      StockLevelsReport: $resource(url + '/reports/inventories/generate')
```

Comment:
@lauravignoli Please update the resource name to match https://github.com/jembi/bsis/pull/535/commits/9ac3842646e8a315e464ba6b54b76d8aa23ddf2d.